packages/main/src/Menu.ts

```diff
@@ -553,7 +584,7 @@ class Menu extends UI5Element {
 			: (target.getRootNode() as ShadowRoot).host as MenuListItem;
 		const item = menuListItem.associatedItem as MenuItem;
 		const mainMenu = this._findMainMenu(item);
-		mainMenu?.fireEvent("item-focus", { ref: menuListItem, item });
+		mainMenu?.fireEvent<MenuFocusEventDetail>("item-focus", { ref: menuListItem, item }, true, false);
```

Comment:
This is new, I'm expecting just the documentation addition without any changes in implementation. Everybody has tested the code without these event parameters. Seems risky to add these here. If you want it to not bubble, let's have this in a separate change for 1.24

Reply:
done